app/webutil.js

```diff
@@ -121,22 +121,22 @@ export function initSettings (callback /*, ...callbackArgs */) {
             }
         });
     } else {
-        // No-op
+        settings = {};
         if (callback) {
             callback.apply(this, callbackArgs);
         }
     }
 };
 
 // No days means only for this browser session
-export function writeSetting (name, value) {
+export function writeSetting (name, value, temporary) {
```

Comment:
"write" and "temporary" seems like a confusing mix to me. Perhaps introduce a new `setSetting()` that only updates the `settings` object?